.github/workflows/documentation.yml

```diff
@@ -66,8 +66,7 @@ jobs:
             tools/generate_docs.sh build
   lint:
     name: Lint documentation
-    needs: documentation_changes
-    if: ${{ needs.documentation_changes.outputs.documentation == 'true' }}
+    # No dependencies as we have markdown files in a lot of places
```

Comment:
The change is okay for me. Do we still need the job `documentation_changes` with this change?

Reply:
Yes, I think we still need that as the `deploy` job depends on it.

Reply:
You are right. I have overlooked that dependency.